src/buspirate.c

```diff
@@ -76,14 +76,14 @@ struct pdata
 	unsigned char pin_dir;		/* Last written pin direction for bitbang mode */
 	unsigned char pin_val;		/* Last written pin values for bitbang mode */
 	int     unread_bytes;		/* How many bytes we expected, but ignored */
+        int     flag;
```

Comment:
@stefanrueger  I'm looking through the code to see if things look alright. Would be great if you would fix the indentation on this line (buspirate.c, line 79)!

Reply:
Ahh, I used 8 spaces to align, not a tab, so hopefully will look OK in source? Git diff probably thinks a tab is 4 spaces.

Reply:
Well, it will depend on the text editor settings. I personally prefer indents to be two characters,  so in the case above it won't look right. It's probably best to either replace all tabs in the struct with spaces, or use tab instead to match the rest of the code.

Reply:
OK, good point, fixed.

Normally I try to adapt spacing of small changes to existing code (which I failed to do here).

[For new code segments I will normally use my own "handwriting" --- even at the risk of a source file becoming inconsistent in style --- but I rarely change spacing/style of existing code lest reviews and later commit diffs see too many lines without actual content change.]